mobile/src/main/res/values-de/strings.xml

```diff
@@ -123,6 +123,7 @@
     <string name="error_http_code_495">Ungültiges Client-Zertifikat (HTTP-Statuscode 495)</string>
     <string name="error_http_code_496">Client-Zertifikat wird gefordert (HTTP-Statuscode 496)</string>
     <string name="error_http_code_502">Ihr Reverse-Proxy bekam eine ungültige Antwort von openHAB (HTTP-Statuscode 502)</string>
+    <string name="error_http_code_503">openHAB startet gerade (HTTP-Statuscode 503)</string>
```

Comment:
Hmm, that's not entirely correct. That may be one of the possible problems why the 503 http status code is sent, but it's not the only one. What happens if the user is connecting to a remote, which proxy is not available because of too high load and sents a 503 status code instead. The openHAB instance in the back may still be up and is not starting, so the string would be false.

Reply:
True, I'll change the strings.